src/sentry/tasks/unmerge.py

```diff
@@ -216,7 +216,7 @@ def migrate_events(
 
     event_id_set = set(event.id for event in events)
```

Comment:
Since this is a SnubaEvent not Event now, id is the same as event_id. I'm not sure if accessing the value using event_id rather than id field is the preferred way though.

Reply:
You are right. At this point it does not really matter, I think, once we got rid of postgres events, we should remove one of the fields. 
I think in this case using event_id would make the code more readable. Since we do not have type hints here, by reading this method, you do not know immediately whether it is a SnubaEvent or not, thus it is hard to ensure you can make the assumption that id and event_id will be the same.
If you use event_id things are more clear.

Reply:
Ok, updated